PWGLF/TableProducer/LFTreeCreatorNuclei.cxx

```diff
@@ -65,6 +65,7 @@ struct LfTreeCreatorNuclei {
   Configurable<float> nsigmacutLow{"nsigmacutLow", -8.0, "Value of the Nsigma cut"};
   Configurable<float> nsigmacutHigh{"nsigmacutHigh", +8.0, "Value of the Nsigma cut"};
   Configurable<int> trackSelType{"trackSelType", 0, "Option for the track cut: 0 isGlobalTrackWoDCA, 1 isGlobalTrack"};
+  Configurable<int> nITSInnerBarrelHits{"nITSInnerBarrelHits", 1, "Option for ITS inner barrel hits maximum: 3"};
```

Comment:
I would leave `0` as the default so that we don't go automatically in conflict with the `globalTrack` requirement

Reply:
Hi @njacazio. I have now changed it to 0.